### DISCUSSION TITLE
Features extraction

### DISCUSSION THREAD

**Discussion by feersantana5** [isl-org/Open3D#3706] [Q&A] [answered]
is there any possibility to extract objects using their characteristics (iss, fpfh...)?

**Comment by masterchop** [comment]
not sure, i understood your question. but you might want to check open3d-ml

**Comment by feersantana5** [comment]
> not sure, i understood your question. but you might want to check open3d-ml

I can't use open3d-ml in my device due to system requirements, I'm looking for other possibility to extract object using open3d probably with other software

**Comment by ssheorey** [comment, marked as answer]
Hi @feersantana5 ISS is supported in Open3D. Please see this tutorial for Python (Analogous C++ APIs are present as usual):

http://www.open3d.org/docs/latest/tutorial/Advanced/iss_keypoint_detector.html

FPFH is here:

http://www.open3d.org/docs/latest/python_api/open3d.registration.compute_fpfh_feature.html
